en/components/grids_templates/column_selection.md

```diff
@@ -0,0 +1,166 @@
+@@if (igxName === 'IgxGrid') {
+---
+title: Angular Grid Column Selection | Ignite UI for Angular | infragistics
+_description: Learn how to configure column selection with the Angular Material table. The column selection makes the interaction with the Grid more appealing than ever.
+_keywords: column selection, igniteui for angular, infragistics
+---
+}
+@@if (igxName === 'IgxTreeGrid') {
+---
+title: Angular Tree Grid Column Selection | Ignite UI for Angular | infragistics
+_description: Learn how to configure column selection with the Angular Material table. The column selection makes the interaction with the Tree Grid more appealing than ever.
+_keywords: column selection, igniteui for angular, infragistics
+---
+}
+@@if (igxName === 'IgxHierarchicalGrid') {
+---
+title: Angular Hierarchical Grid Column Selection | Ignite UI for Angular | infragistics
+_description: Learn how to configure column selection with the Angular Material table. The column selection makes the interaction with the Hierarchical Grid more appealing than ever.
+_keywords: column selection, igniteui for angular, infragistics
+---
+}
+
+### @@igComponent Column Selection
+The **column selection feature** provides an easy way to select an entire column with a one click action. It **emphasizes** the importance of a particular column by focusing on the header cell(s) and everything below. The feature comes with a rich [`API`]({environment:angularApiUrl}) that allows manipulation of the **state**, **data extraction** of the **selected** fractions easing **data analysis** operations and visualizations.
+
+<div class="sample-container loading" style="height:570px">
+    <iframe id="grid-column-selection" data-src='{environment:demosBaseUrl}/grid/grid-column-selection' width="100%" height="100%" seamless frameborder="0" class="lazyload no-theming"></iframe>
+</div>
+<br/>
+<div>
+<button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="grid-column-selection-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+<div class="divider--half"></div>
+
+### Basic usage
+
+#### Interactions
+All of the presented columns are [`selectable`]({environment:angularApiUrl}/classes/igxcolumncomponent.html#selectable) by default. With that being said, in order to select a column, we just need to click over the targeted one to mark it as [`selected`]({environment:angularApiUrl}/classes/igxcolumncomponent.html#selected).
```

Comment:
in order to select a column, we just need to click on the targeted on, which mark it as [`selected`]({environment:angularApiUrl}/classes/igxcolumncomponent.html#selected).